packages/playground/src/dashboard/components/public_ips_table.vue

```diff
@@ -40,9 +40,9 @@
         <v-toolbar color="primary" class="custom-toolbar">
           <p class="mb-5">Delete IP</p>
         </v-toolbar>
-        <v-card-text> Are you sure you want to delete IP {{ itemToDelete?.ip }}? </v-card-text>
+        <v-card-text class="text-subtitle-1"> You sure you want to delete IP {{ itemToDelete?.ip }}? </v-card-text>
```

Comment:
The question shouldn't start with `you`, imo the `are` should be re-added or it should be rephrased to `Do you want to delete this IP?`

Reply:
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/10484295/5f09dd3a-bff8-4b12-89eb-a9393c43a6ee)
fixed

also the typo